packages/use-query-params/README.md

```diff
@@ -9,10 +9,11 @@ $ pnpm add @scaleway/use-query-params react-router-dom
 ```
 
 ## Usage
+
 ```js
 import React from 'react'
```

Comment:
Is this react import mandatory?

Reply:
We don't know if our user use new jsx transform ?